src/components/Home/Aditya/index.tsx

```diff
@@ -17,6 +17,11 @@ const Aditya: React.FC = () => {
             Week-3
           </button>
         </Link>
+        <Link to='week4'>
+          <button className={`weekButton ${selectedWeek === 'week4' ? 'active' : ''}`} value='week4' onClick={handleWeekClick}>
+            Week-4
+          </button>
+        </Link>
       </div>
       <Outlet />
     </AdityaWrapper>
```

Comment:
use another name instead of using your self name in the component ,it may be written as mainWrapperWeek3 or what ever the week it is.
(It was suggested by Harendra)